cob_hardware_config/xacrodisplay.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+  <arg name="gui" default="false" />
+  <arg name="rvizconfig" default="$(find urdf_tutorial)/urdf.rviz" />
```

Comment:
these introduces an `exec_depend>` to `urdf_tutorial`...
how about copying that file...or simply use `cob_hardware_config/robots/$(arg robot)/$(arg robot).rviz` (should be fine enough having a RobotModel loaded to the displays...